configure

```diff
@@ -787,6 +787,9 @@ def configure_arm(o):
 
   o['variables']['arm_fpu'] = options.arm_fpu or arm_fpu
 
+  if flavor == 'win':
+    o['msvs_windows_sdk_version'] = 'v10.0'
```

Comment:
Haha yeah, I got thrown off by that too- it seemed like a roundabout way to do things 😄